assets/js/googlesitekit/data/create-notifications-store.js

```diff
@@ -63,11 +67,11 @@ export const createNotificationsStore = ( type, identifier, datapoint, {
 	const STORE_NAME = storeName || `${ type }/${ identifier }`;
 
 	const INITIAL_STATE = {
-		serverNotifications: undefined,
+		serverNotifications: server ? undefined : null,
 		// Initialize clientNotifications as undefined rather than an empty
 		// object so we can know if a client notification was added and then
 		// removed from state.
-		clientNotifications: undefined,
+		clientNotifications: client ? undefined : null,
```

Comment:
I think this should not be `null`, but an empty object if `client` is `false`. Otherwise, in the `getNotifications` selector, the code will always try to destructure `null`. cc @tofumatt

Reply:
Ah, good point. I'm good with that change 👍

Reply:
Fixed